packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -282,7 +299,15 @@ export class ProjectService {
     }
 
     async delete(projectUuid: string, user: SessionUser): Promise<void> {
-        if (user.ability.cannot('delete', 'Project')) {
+        const { organizationUuid } =
+            await this.projectModel.getWithSensitiveFields(projectUuid);
```

Comment:
We can use the get method without sensitive fields, just for safety.

Reply:
I was doing that first, but then I realized the `get` method is calling `getWithSensitiveFields` with some filtering, so I thought this was more straightforward and efficient for this purpose. But I'm happy to change it if you prefer